app/controllers/programs_controller.rb

```diff
@@ -69,83 +68,83 @@ def create
       # there should only be one sub-program immediately after a program create
       @sub_program = @program.sub_programs.first
       redirect_to action: :index
-       flash[:notice] = "New program \"<a href=\"#{edit_program_sub_program_path(@program, @sub_program)}\">#{@program.name}</a>\" created"
+      flash[:notice] = "New program \"<a href=\"#{edit_program_sub_program_path(@program, @sub_program)}\">#{@program.name}</a>\" created"
     else
-      flash[:error] = "Please review the form problems below."
+      flash[:error] = 'Please review the form problems below.'
       render :new
     end
   end
 
   private
```

Comment:
I think we have been trying to move the `private` to the individual methods? RubCop doesn't like the indentation, and I think it is too hard to notice the keyword just sort of hanging out here...